react/src/lib/Popover/index.js

```diff
@@ -216,8 +266,39 @@ class Popover extends React.Component {
               },
               onMouseLeave: e => {
                 e.persist();
-                this.setState({ isHovering: false }, () => this.delayedHide(e));
+                this.handleHide(e);
               },
+              onKeyDown: (e) => {
+                if (this.state.isOpen && this.overlay && this.anchorRef) {
+                  const eventOverlay = ReactDOM.findDOMNode(this.overlay);
+                  const trigger = ReactDOM.findDOMNode(this.anchorRef);
+                  const tabbableElements = eventOverlay.querySelectorAll('[tabindex="0"]');
+
+                  switch (e.which) {
```

Comment:
Can you add some code comments as to what/why this is doing? If I stumbled on to this code, I wouldn't know how or what to update most likely